packages/fury/lib/fury.js

```diff
@@ -268,7 +268,10 @@ class Fury {
     const promise = adapter.serialize({ api, namespace: this.minim, mediaType });
 
     if (done) {
-      promise.then(result => done(null, result), done);
+      promise
```

Comment:
Why did you change this?

Reply:
It was needed, otherwise there were a lot of warnings getting printed.

Reply:
Can you perhaps explain the difference a bit further or give me some steps I can reproduce these warnings?

There's other cases where this syntax is used

Reply:
```
(node:10542) UnhandledPromiseRejectionWarning: AssertionError: expected 'FORMAT: 1A\n\n# My API\n\n## Group A\n\n### /message\n\n#### GET\n\n+ Response 200 (text/plain)\n\n        Hello, World!' to deeply equal 'FORMAT: 1A\n\n# My API\n\n## Group A\n\n### /message\n\n#### GET\n\n+ Response 200 (text/plain)\n\n        Hello, World'
    at /Users/pksunkara/Coding/apiary/api-elements.js/packages/fury-adapter-apib-serializer/test/adapter-test.js:46:43
    at /Users/pksunkara/Coding/apiary/api-elements.js/packages/fury/lib/fury.js:271:30
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
(node:10542) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10542) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```